src/main/java/at/hannibal2/skyhanni/features/hunting/ShardTrackerDisplay.kt

```diff
@@ -0,0 +1,196 @@
+import at.hannibal2.skyhanni.utils.ItemUtils.getInternalName
+import at.hannibal2.skyhanni.utils.ItemUtils.repoItemNameCompact
+import at.hannibal2.skyhanni.utils.KeyboardManager
+import at.hannibal2.skyhanni.utils.KeyboardManager.isKeyHeld
+import at.hannibal2.skyhanni.utils.NeuInternalName
+import at.hannibal2.skyhanni.utils.NeuInternalName.Companion.toInternalName
+import at.hannibal2.skyhanni.utils.OSUtils
+import at.hannibal2.skyhanni.utils.RenderDisplayHelper
+import at.hannibal2.skyhanni.utils.RenderUtils.renderRenderables
+import at.hannibal2.skyhanni.utils.SkyBlockUtils
+import at.hannibal2.skyhanni.utils.compat.stackUnderCursor
+import at.hannibal2.skyhanni.utils.renderables.Renderable
+import at.hannibal2.skyhanni.utils.renderables.ScrollValue
+import at.hannibal2.skyhanni.utils.renderables.primitives.text
+import org.lwjgl.input.Keyboard
+import java.util.zip.GZIPInputStream
+import kotlin.io.encoding.Base64
+import kotlin.io.encoding.ExperimentalEncodingApi
+
+@SkyHanniModule
+object ShardTrackerDisplay {
+
+    val config get() = SkyHanniMod.feature.hunting.shardTracker
+    private fun isEnabled() = SkyBlockUtils.inSkyBlock && config.enabled
+
+    init {
+        RenderDisplayHelper(
+            outsideInventory = true,
+            inOwnInventory = true,
+            condition = { isEnabled() },
+            onRender = {
+                render()
+            },
+        )
+    }
+
+    private var renderables: List<Renderable>? = null
+    private val trackedShards = mutableMapOf<String, Int>()
+
+    private fun toggleShard(neuId: NeuInternalName) {
+        if (!AttributeShardsData.isAttributeShard(neuId)) {
+            ErrorManager.logErrorStateWithData(
+                "Error Getting Attribute Shard",
+                "$neuId is not a valid attribute shard"
+            )
+        }
+        val id = neuId.asString()
+        if (trackedShards.contains(id)) {
+            trackedShards.remove(id)
+        } else {
+            trackedShards[id] = -1
+        }
+    }
+
+    val scroll = ScrollValue()
+
+    fun render() {
+        if (!isEnabled()) return
+        if (trackedShards.isEmpty()) {
+            renderables = null
+            return
+        }
+        val invCurrentlyOpen = InventoryUtils.inAnyInventory()
+        val renderable = mutableListOf<Renderable>()
+        for (shard in trackedShards) {
+            val shardId = shard.key.toInternalName()
+            val shardDisplayName = shardId.repoItemNameCompact.replace(" Shard", "")
+
+            val shardName = AttributeShardsData.shardInternalNameToShardName(shardId)
+            val amountInHuntingBox = AttributeShardsData.getAmountInHuntingBox(shardName)
+            val amountUntilMax = if (shard.value == -1) AttributeShardsData.getAmountUntilMax(shardName) else shard.value
+
+            if (amountUntilMax == 0) {
+                renderable.add(Renderable.text("$shardDisplayName§7: §a$amountInHuntingBox"))
```

Comment:
Looks a bit weird to have all the shards in line with the title imo; item tracker items have a space in front of them
```suggestion
                renderable.add(Renderable.text(" $shardDisplayName§7: §a$amountInHuntingBox"))
```